src/lib/index.ts

```diff
@@ -143,24 +143,22 @@ export class GoogleToken {
    * @param callback The callback function.
    */
   public revokeToken(callback: (err?: Error) => void): void {
-    if (this.token) {
-      request(GOOGLE_REVOKE_TOKEN_URL + this.token, (err: Error) => {
-        if (err) {
-          callback(err);
-          return;
-        }
-        this.configure({
-          email: this.iss,
-          sub: this.sub,
-          key: this.key,
-          keyFile: this.keyFile,
-          scope: this.scope
-        });
-        callback();
-      });
-    } else {
+    if (!this.token) {
       setImmediate(callback, new Error('No token to revoke.'));
+      return;
     }
+    axios.get(GOOGLE_REVOKE_TOKEN_URL + this.token)
+        .then(r => {
+          this.configure({
+            email: this.iss,
+            sub: this.sub,
+            key: this.key,
+            keyFile: this.keyFile,
+            scope: this.scope
+          });
+          callback();
+        })
+        .catch(e => callback);
```

Comment:
This smells fishy to me. Did you mean `.catch(callback)`?

Perhaps we are missing a test.